packages/dates/src/utilities/formatDate.ts

```diff
@@ -14,6 +14,7 @@ export function memoizedGetDateTimeFormat(
 
 const browserFeatureDetectionDate = Intl.DateTimeFormat('en', {
   hour: 'numeric',
+  calendar: 'gregory',
```

Comment:
It would be a nice to add comment about why this restriction is being added so that we can remember to remove it once there's wider support.